apps/webapp/Dockerfile

```
@@ -63,5 +61,4 @@
 COPY --from=builder ./app/out/full/ .
 RUN pnpm run build
 
-CMD cd /app/apps/webapp/ && pnpm run start
-
+CMD cd /app/apps/webapp/ && pnpm run start
```

Hadolint (2.12.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)